### PR TITLE
Feature work with husky (#47)

### PR DESCRIPTION
**Pull request by matt-degraffenreid**
feat(githook.ts): add support for git core hooks path and fallback to default symlink url if not found

feat(utils/git.ts): add getCoreHooksPath function to retrieve the path of the core hooks directory.

### CODE REVIEW

**Comment by matt-degraffenreid** [comment]
Pull request for #47

File: src/commands/githook.ts [hunk 1]
```diff
@@ -47,6 +62,7 @@ export const hookCommand = command(
           );
         }
 
+        console.log("made it")
```
Comment (AdrielBento) [inline src/commands/githook.ts:65]: @matt-degraffenreid I think you forgot to remove that log
```suggestion

```

Comment (matt-degraffenreid) [reply src/commands/githook.ts]: hate when that happens, good eye. just update that.

**Comment by matt-degraffenreid** [comment]
Noticing some undesired behavior will need another commit to fix

**Comment by di-sukharev** [comment]
changed base branch to dev

**Review by di-sukharev — APPROVED** [review]

**Comment by di-sukharev** [comment]
@matt-degraffenreid bro, awesome PR, let's merge when you solve the conflicts :)

**Comment by di-sukharev** [comment]
@matt-degraffenreid bro, could you also add this fix for windows https://github.com/di-sukharev/opencommit/issues/11#issuecomment-1483977918

**Comment by matt-degraffenreid** [comment]
Fixed the issue I had with behavior there was a flag missing from the hook process causing all files to get staged. Copy and pasted the code from the commit process to make the flag. Will take a look at number #11 and fix the merge conflicts

**Comment by matt-degraffenreid** [comment]
Resolved the conflicts, and addressed #11. Just need a reviewer and this one is good to merge into dev

**Comment by matt-degraffenreid** [comment]
Making a new pull request fubared this one

**Comment by jefnic23** [comment]
description=true

**Pull request closed without merging**